paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/UpdatedDataFieldsProcessFunctionBase.java

```diff
@@ -214,6 +218,10 @@ protected List<SchemaChange> extractSchemaChanges(
             oldFields.put(oldField.name(), oldField);
         }
 
+        boolean allowTypeChange =
+                this.typeMapping == null
+                        || !this.typeMapping.containsMode(TypeMapping.TypeMappingMode.NO_CHANGE);
+
```

Comment:
Where the "DECIMAL_NO_CHANGE" use?

Reply:
Thanks for review, I'll check that again.